media-gfx/zbar/zbar-0.23.1.ebuild

```diff
@@ -193,6 +193,8 @@ multilib_src_configure() {
 		use test && myeconfargs+=( --without-zbarimg-tests )
 	fi
 
+	# use bash (bug 721370)
+	CONFIG_SHELL="${BASH}" \
```

Comment:
Does this actually work? The bug had an example to just call `/bin/bash`, wouldn't that be better?

Reply:
I don't think it makes a difference :)